Core/Graphics/Include/Babylon/Graphics.h

```diff
@@ -16,9 +16,15 @@ namespace Babylon
         template<typename... Ts>
         static std::unique_ptr<Graphics> CreateGraphics(Ts...);
 
+        template<typename... Ts>
+        static std::unique_ptr<Graphics> CreateGraphicsFromContext(Ts...);
+
         template<typename... Ts>
         void UpdateWindow(Ts...);
 
+        template<typename... Ts>
+        void UpdateContext(Ts...);
```

Comment:
Maybe once the new pattern comes in, it might make sense to consolidate this, `UpdateWindow`, and other functions with related purposes into a `Reinitialize(const GraphicsConfiguration&)` method, or something like that.